dm/ctl/common/util.go

```diff
@@ -100,19 +117,38 @@ func (c *CtlClient) sendRequest(ctx context.Context, reqName string, req interfa
 
 // SendRequest send request to master.
 func SendRequest(ctx context.Context, reqName string, req interface{}, respPointer interface{}) error {
-	err := ctlClient.sendRequest(ctx, reqName, req, respPointer)
-	if err == nil || status.Code(err) != codes.Unavailable {
+	err := GlobalCtlClient.sendRequest(ctx, reqName, req, respPointer)
+	if err == nil {
+		return nil
+	}
+	var opts []interface{}
+	switch status.Code(err) {
+	case codes.ResourceExhausted:
+		matches := re.FindStringSubmatch(err.Error())
+		if len(matches) == 3 {
+			msgSize, err2 := strconv.Atoi(matches[1])
```

Comment:
Should we increase msgSize a little more?

Reply:
unit test shows this is enough

Reply:
Will the result become a little bigger for the second request?

Reply:
we can't decide a reasonable delta 🤔

Reply:
OK, I'm just afraid it's too strict sometimes.

Reply:
Is there a hard limit of gRPC message size? And how about use next_pow_of_2(msg_size) as the new limit

Reply:
> Is there a hard limit of gRPC message size?

MaxInt. This only has memory affects. next_pow_of_2 seems more like a strategy for container capacity growing, while here I think using exactly the size is OK, because we sent the RPC again in a short time so the response is more likely not changed.